string node_name
string node_type
bool cancel
Advertisement[] advertisements
---
bool success
